### ISSUE TITLE
Images Example: Paste URL seems to be broken

### ISSUE THREAD

**Issue by dons20** [ianstormtaylor/slate#2626]
#### Do you want to request a _feature_ or report a _bug_?

Bug Report

#### What's the current behavior?

Following the instructions in this example, I tried to paste an image URL but it didn't add the image. However, it does work when clicking the image icon and pasting in the prompt that follows.

Source: https://www.slatejs.org/#/images

Bug in action: ![Bug](https://thumbs.gfycat.com/TartCornyHorseshoecrab-size_restricted.gif)

#### What's the expected behavior?

Image should be pasted if it's a valid URL.

**Comment by brendancarney** [comment]
Looks like it's failing here: https://github.com/ianstormtaylor/slate/blob/master/examples/images/index.js#L40

**Comment by brendancarney** [comment]
Just saw there is an open PR for this: https://github.com/ianstormtaylor/slate/pull/2592

**Comment by dons20** [comment]
I was about to submit a PR as well. What do you think about having this instead? @brendancarney 

```
function isImage(url) {
  return !!imageExtensions.find((e) => { return url.includes(e) })
}
```

**Comment by brendancarney** [comment]
@dons20 since someone has already started work on it, it's probably best to let them finish. 

Thanks for the willingness though, and if you'd like to make a PR, there are many issues in the repo that could use some help!

**Comment by dons20** [comment]
@brendancarney sounds good, I'll see if I can help out on some other issues 😁

**Comment by dons20** [comment]
Fixed as per #2592
